### DISCUSSION TITLE
Tile size is too big

### DISCUSSION THREAD

**Discussion by PKASHYAP-BEC** [iTwin/itwinjs-core#5541] [Q&A]
Hi , is anyone facing a issues where in initial load we have a tile size >200mb in zip and ~900mb on disk. 

can someone suggest to overcome this behavior , we like to have a cap on sizing for not more that 2 mb.

thanks in advance for any suggestion.

link of tile https://imodelbackendprodeussa01.blob.core.windows.net/14cbaaab-8a77-4cb6-b546-553c0a5cf2e0/tiles/1e_d-E%3A0_0x30000000001/48a2db37-92aa-436e-b29a-48bb89464d82_6cf9ab1405ca6511/-b-1-0-1-1-1

![image](https://github.com/iTwin/itwinjs-core/assets/60220061/7dab523a-8f61-4a5e-a0bd-e29b1349d8ba)

**Comment by pmconne** [comment]
> Public access is not permitted on this storage account. 

That's an unexpectedly massive tile. Without more information I can't say why it's so big.
There is no mechanism by which you can cap tile sizes to 2mb.

**Reply by PKASHYAP-BEC** [reply]
yes , that is strange , it is model is hosted on itwin platform and so are tiles .

please let me know what information can help to reduce tile sizes

**Comment by spowell-216** [comment]
@pmconne I might be able to elaborate on what we are experiencing.  We have an iModel 49855 spatial elements, 19121 are concentrated in the circled area.  On initial load everything renders except the circled area and we are left with a waiting on 1 tile, 20-30seconds depending on network speed.  
![Screenshot 2023-05-18 093823](https://github.com/iTwin/itwinjs-core/assets/51784707/21848a72-6111-47b7-8124-1f31d9ba260f)
The geometry in question is on a separate bis.physicalpartition and originated from the Revit Connector.  I think it might contain a considerable amount of BREP geometry, but hard to confirm.  We are looking for guidance on how to troubleshoot or some optimizations we can look into for tile generation.

**Comment by pratikshan85** [comment]
@pmconne do you have any comment on the latest question by @spowell-216 ?
shall we need to connect with Revit connector dev team for the issue?

**Reply by pmconne** [reply]
The most productive way to proceed would involve getting access to the iModel. @pratikshan85 can you engage with @spowell-216 over private channels?

**Reply by spowell-216** [reply]
@pratikshan85 let me know the best way to engage privately.  Thanks.

**Reply by pratikshan85** [reply]
@spowell-216 Can you please provide your email address so that we can discuss this over a private channel?

**Reply by spowell-216** [reply]
@pratikshan85 please use spowell@bechtel.com
